auth/api/grpc/client.go

```diff
@@ -129,7 +138,7 @@ func (client grpcClient) Authorize(ctx context.Context, req *mainflux.AuthorizeR
 	ctx, close := context.WithTimeout(ctx, client.timeout)
 	defer close()
 
-	res, err := client.authorize(ctx, authReq{Act: req.Act, Obj: req.Obj, Sub: req.Sub})
+	res, err := client.authorize(ctx, authReq{Act: req.GetAct(), Obj: req.GetObj(), Sub: req.GetSub()})
```

Comment:
Yes, I agree. But those getter functionalities come from auto-generated proto files. Therefore, I've used them instead of calling struct members (as `res.Obj`) directly. . Should it be called `res.Obj` instead of `res.GetObj()`?